packages/salesforcedx-utils-vscode/src/helpers/activationTracker.ts

```diff
@@ -24,7 +24,7 @@ export class ActivationTracker {
     };
   }
 
-  async markActivationStop(activationEndDate?: Date): Promise<void> {
+  public async markActivationStop(activationEndDate?: Date): Promise<void> {
```

Comment:
when adding these, I looked around for usage.  if I couldn't find any, I marked them private.  Most needed to go to public.